Essentials/src/main/java/com/earth2me/essentials/utils/VersionUtil.java

```diff
@@ -248,12 +257,12 @@ public boolean equals(final Object o) {
                 minor == that.minor &&
                 patch == that.patch &&
                 revision == that.revision &&
-                prerelease == that.prerelease;
+                    preRelease == that.preRelease;
```

Comment:
no

Reply:
This thing doesn't work with Spigot 1.18-rc3-R0.1-SNAPSHOT